### PR TITLE
feat: increase supply caps for ETH and WBTC for the linea quest

### PR DESCRIPTION
**Pull request by Julink-eth**
The Linea quest is having issues right now because we reach the supply limits, I created this migration script to increase them.

### CODE REVIEW

File: deployments/linea-goerli/usdc/migrations/1687878801_increase-supply-caps.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  async enacted(deploymentManager: DeploymentManager): Promise<boolean> {
+    return false;
+  },
+
+  async verify(
+    deploymentManager: DeploymentManager,
+    govDeploymentManager: DeploymentManager,
+    preMigrationBlockNumber: number
+  ) {
+    await deploymentManager.spider(); // We spider here to pull in Linea COMP now that reward config has been set
```
Comment (scott-silver) [inline deployments/linea-goerli/usdc/migrations/1687878801_increase-supply-caps.ts:92]: I don't think this line is necessary in this situation

File: deployments/linea-goerli/usdc/migrations/1687878801_increase-supply-caps.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    const goerliActions = [
+      // 1. Set Comet configuration
+      {
+        contract: lineaMessageService,
+        signature: 'sendMessage(address,uint256,bytes)',
+        args: [bridgeReceiver.address, 0, l2ProposalData],
+      },
+    ];
+
+    const description =
```
Comment (scott-silver) [inline deployments/linea-goerli/usdc/migrations/1687878801_increase-supply-caps.ts:69]: nit; can be on a single line

File: deployments/linea-goerli/usdc/migrations/1687878801_increase-supply-caps.ts (hunk 1 above)
Comment (Julink-eth) [reply deployments/linea-goerli/usdc/migrations/1687878801_increase-supply-caps.ts]: Done

**Review by kevincheng96 — APPROVED** [review]
LGTM

**Pull request merged**